lib/chef/node_map.rb

```diff
@@ -78,6 +78,12 @@ def get(node, key)
       nil
     end
 
+    def clear
```

Comment:
Actually, it looks like I didn't need it (searching for "clear" in the patch didn't turn anything up.  Cruft from an earlier revision perhaps.